projects/bbb_canbus_listener/main.cpp

```diff
@@ -203,9 +203,15 @@ int main(int argc, char **argv) {
 
                 std::map<std::string, NetworkTable::Value> values;
                 NetworkTable::Value gps_longitude;
-                double longitude = GET_GPS_LONG(frame.data);
+
+                // Perform a unit conversion from decimal degrees minutes to degrees.
+                double longitudeDDM = GET_GPS_LONG(frame.data);
+                int longitudeDD = longitudeDDM / 100;
+                double longitudeM = longitudeDDM - longitudeDD*100;
+                double longitude_degrees = longitudeDD + longitudeM/60;
+
                 gps_longitude.set_type(NetworkTable::Value::INT);
```

Comment:
Can you change the type to NetworkTable::Value::FLOAT

This is our google protobuf object - it can either be a int, float, string, boat object, or waypoint data